pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -12,6 +12,7 @@ theme: responsive
 - js_locale = request.locale.to_s.downcase.tr('-', '_')
 %script{src: webpack_asset_path("js/#{js_locale}/common_locale.js")}
 
+%link{href: "/shared/css/barlowSemiCondensed-font.css", rel: "stylesheet"}
```

Comment:
Should this link also be `/shared/css/barlow-semi-condensed-font.scss` since you renamed it?

Reply:
True, my mistake. Thanks a lot for noticing!
Fixed